ops/envs/batches.env

```diff
@@ -3,7 +3,7 @@ ADDRESS_MANAGER_ADDRESS=
 DEBUG=info*,error*,warn*,debug*
 
 MAX_L1_TX_SIZE=90000
-MIN_L1_TX_SIZE=0
+MIN_L1_TX_SIZE=32
```

Comment:
32 bytes seemed like a reasonable default minimum.
Appended state batches obviously requires at least 32 bytes. 
Appending a sequencer batch with zero contexts can be done with just a `uint40`, but including a `BatchContext` would put the data size well over 32.